### ISSUE TITLE
[Bug]: Crash when updating series sequence

### ISSUE THREAD

**Issue by YodaDaCoda** [advplyr/audiobookshelf#1919]
### Describe the issue

From the series view, I tried to edit a book's metadata (adding a series number to a book that was missing it). Upon clicking 'Save' I get a red toast saying "Socket disconnected" and an error in the logs, then the server crashes (and the container reboots).

```
audiobookshelf    | [2023-07-17 12:04:29] DEBUG: [BookMetadata] Key updated series [
audiobookshelf    |   {
audiobookshelf    |     displayName: 'Cradle #2',
audiobookshelf    |     id: '3a1118d2-0b5e-46b7-bac3-5e1096e1ad78',
audiobookshelf    |     name: 'Cradle',
audiobookshelf    |     sequence: '2'
audiobookshelf    |   }
audiobookshelf    | ] (BookMetadata.js:211)
audiobookshelf    | [2023-07-17 12:04:29] DEBUG: [LibraryItemController] Updated library item media 2 Soulsmith (LibraryItemController.js:144)
audiobookshelf    | /node_modules/sequelize/lib/model.js:2413
audiobookshelf    |         throw new Error("You attempted to save an instance with no primary key, this is not allowed since it would result in a global update");
audiobookshelf    |               ^
audiobookshelf    |
audiobookshelf    | Error: You attempted to save an instance with no primary key, this is not allowed since it would result in a global update
audiobookshelf    |     at BookSeries.save (/node_modules/sequelize/lib/model.js:2413:15)
audiobookshelf    |     at BookSeries.update (/node_modules/sequelize/lib/model.js:2598:23)
audiobookshelf    |     at Function.fullUpdateFromOld (/server/models/LibraryItem.js:223:52)
audiobookshelf    |     at async ApiRouter.updateMedia (/server/controllers/LibraryItemController.js:145:7)
```

### Steps to reproduce the issue

I haven't tried to reproduce on other series/views.

1. Open the Series view for a series
2. Click the pencil icon to edit book details
3. Hover over the Series entry and click the pencil icon to edit
4. Add a series number to the book
5. Press 'Submit'
6. Press 'Save' <- error occurs

### Audiobookshelf version

v2.3.1

This also occurred on v2.3.0, but I didn't check earlier versions.

### How are you running audiobookshelf?

Docker

**Comment by bebopjunky** [comment]
This happened to me as well

**Comment by Nab0y** [comment]
Confirmed:
1. Added a new book, and it appeared in the "Recently Added" section.
2. Submitted the book for editing for the first time, and everything is fine.
3. Applied the changes made, including the series number and reader, and everything is still fine.
4. Submitted the book for editing again from the same "Recently Added" section, and the server crashed, causing Docker to stop. There are no logs available for the data.

**Comment by Nab0y** [comment]
I think I have found the crash logs.

![image](https://github.com/advplyr/audiobookshelf/assets/13904687/d4149fc9-78c5-493b-8a67-c9c8cdf22619)

**Comment by warwolf** [comment]
I can add a bit to this.

While updating from Audible every normal book works fine even if i cannot update manually.
However if the book got a sequence range ex for book 1-5 it crashes when update from Audible.

**Comment by advplyr** [comment]
Got this one. I'll put out another small patch later today.

**Comment by advplyr** [comment]
Fixed in [v2.3.2](https://github.com/advplyr/audiobookshelf/releases/tag/v2.3.2)
